### ISSUE TITLE
Scheduler diverges at high rate limits

### ISSUE THREAD

**Issue by krfall** [NetSys/bess#534]
Using a simple src->sink topology with a large rate limit works properly up to some limit, and then then hits a point where it no longer tracks properly.

More specifically, on 2.6GHz cpu, up to a 68Gb/s RL is fine but a setting of 69Gb/s gives about 280Mb/s.

Heres is some "show tc" output for these two cases, respectively:
```
20:36:10.627260          CPU MHz   scheduled        Mpps        Mbps  pkts/sched    cycles/p
--------------------------------------------------------------------------------------------
W0 mylimit              1164.117     3160844     101.147   67970.804      32.000      11.509
W0 !leaf_src1:0         1164.115     3160841     101.147   67970.730      32.000      11.509
--------------------------------------------------------------------------------------------

20:36:29.158229          CPU MHz   scheduled        Mpps        Mbps  pkts/sched    cycles/p
--------------------------------------------------------------------------------------------
W0 mylimit                 9.295       13035       0.417     280.325      32.000      22.282
W0 !leaf_src1:0            9.296       13036       0.417     280.332      32.000      22.283
--------------------------------------------------------------------------------------------
```

**Comment by ddiproietto** [comment]
Maybe this line is the problem?

https://github.com/NetSys/bess/blob/7b5d7a7cd2611ea61f6072b1dbdeb43fb81caeb6/core/traffic_class.h#L504`

`x` is the user input.  If I'm not mistaken, the left shift by `USAGE_AMPLIFIER_POW - 4`, (28) will cause anything greater than `2**36` (68.7G) to overflow

**Comment by sangjinhan** [comment]
Fixed by #542
